backend/models/issue.js

```diff
@@ -525,7 +525,6 @@ schema.statics.getIssuesList = function(dbColOptions, user, branch, revision, id
 
 	const projection = {
 		extras: 0,
-		"comments": 0,
```

Comment:
Why has this been removed?

Reply:
Because we need list of comments for each issues to filter them by text included in comment.

Reply:
The filter should not check the contents of the comments. It is sufficient to only filter the issue details excluding comments. I have confirmed this with @carmenfan.

Reply:
That's weird. 😕 So, why do you have this functionality on staging? Do want to remove it?

Reply:
@simplypixi 

I don't know how react is doing it (@sanmont3drepo is doing the code review), but in the angular version, the issue list shares the data model with issue details

When we fetch the issue list, it doesn't contain comments (as issue list can get overly large, we want to just get the "cliff notes" of each issue, sufficient to show the listing). 

If the user clicked into the issue (i.e. if we've entered the issue detail tab, where full issue details (including comments) are fetched, this updates the data model that is used by both issuelist and issue details

when a user does a string search, we look through that data model. Depending on whether the full issue details has been fetched or not, some issues may have comments loaded and some don't. As of staging, we look through the comments if it's in the datamodel , but if it's not there we don't try to fetch them.

If the data model is separate in the react version (should it though?), I'd remove this feature. I guess the bottom line I want to say is, we don't want to fetch the full issue detail on the issueList fetch API.

Reply:
I understand. We'll implement this the same way as in the angular version. ;)

Reply:
Fixed ;]